build/lib/src/generate/run_builder.dart

```diff
@@ -24,7 +24,8 @@ Future<Null> runBuilder(Builder builder, Iterable<AssetId> inputs,
     AssetReader reader, AssetWriter writer, Resolvers resolvers,
     {Logger logger, String rootPackage}) async {
   logger ??= new Logger('runBuilder');
-  rootPackage ??= new Set.from(inputs.map((input) => input.package)).single;
+  rootPackage ??=
+      new Set<String>.from(inputs.map((input) => input.package)).single;
```

Comment:
I guess that makes sense, although couldn't it just be `new Set.from(stuff as Iterable<String>)` which isn't really much worse than `new Set<String>.from(stuff)`?

Actually though, why not just use `inputs.map((input) => inputs.package).toSet()`, that does the right thing it looks like.